packages/api/cms-api/schema.gql

```diff
@@ -123,6 +123,11 @@ type FilenameResponse {
   isOccupied: Boolean!
 }
 
+type SitePreviewHash {
+  timestamp: Float!
+  hash: String!
+}
+
```

Comment:
This change seems unrelated to me 🤔

Reply:
hmm weird. Seems like a generator change?

Reply:
Should I revert it or just leave it because it basically just reordered it?

Reply:
This will be resolved after merging next into the feature branch, so we may ignore it here.

In general, it's good idea to not add changes which aren't related to your changes (i.e., no `git add .`)